src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1460,7 +1461,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return s_variablesMappings.GetValue(
                     Me,
                     Function(BoundUsing)
-                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration))
+                        Dim usingStatementSyntax = Syntax.ChildNodes().OfType(Of UsingStatementSyntax).SingleOrDefault()
```

Comment:
>Syntax.ChildNodes().OfType(Of UsingStatementSyntax).SingleOrDefault() [](start = 51, length = 69)

It looks like the syntax is always a UsingBlockSyntax, so this could be just ```DirectCast(Syntax, UsingBlockSyntax).UsingStatement```.